packages/teleport/src/services/user/makeUser.ts

```diff
@@ -14,16 +14,26 @@
  * limitations under the License.
  */
 
-import { at } from 'lodash';
 import { User } from './types';
 
 export default function makeUser(json): User {
-  const [name, roles, authType] = at(json, ['name', 'roles', 'authType']);
+  json = json || {};
```

Comment:
```typescript
export default function makeUser(json = {}): User {
  const { ... etc } = etc;
}
```

Also can we get a type for `json`?

Reply:
the `json` param does not require typing. in our `services/*` any `makeXXX` funcs are typically used to process a untyped api response and return it typed (we just call it `json` maybe not the best naming)

Reply:
Can you make it the `any` type then? Not typing it adds more technical debt for if we enable strict mode

Reply:
the `any` sounds reasonable to me

Reply:
Yeah I've been asking for us to type these requests and responses but that'll be a larger longer running undertaking. I've also previously brought up the concern around `json` naming as it's not json 😄 but it's one of those conventions we should discuss and then change all at once.